cmd/podman/run.go

```diff
@@ -66,7 +70,7 @@ func runCmd(c *cli.Context) error {
 	}
 
 	// Gather up the options for NewContainer which consist of With... funcs
-	options = append(options, libpod.WithRootFSFromImage(createConfig.ImageID, createConfig.Image, true))
+	options = append(options, libpod.WithRootFSFromImage(createConfig.ImageID, createConfig.Image, useImageVolumes))
```

Comment:
We should only set useImageVolumes to true if we are doing bindmount image volumes - should not set if we're doing tmpfs, that happens entirely in podman

Reply:
fixed